extensions/sql-migration/src/api/utils.ts

```diff
@@ -865,3 +865,26 @@ export async function getBlobLastBackupFileNamesValues(lastFileNames: azureResou
 	}
 	return lastFileNamesValues;
 }
+
+export async function getBlobCredentialsValue(credentials: SimpleExecuteResult): Promise<CategoryValue[]> {
+	let credentialsValues: CategoryValue[] = [];
+	credentials.rows.forEach((row) => {
+		credentialsValues.push({
+			name: row[0].displayValue,
+			displayName: row[0].displayValue
+		});
+	});
+	if (credentialsValues.length === 0) {
+		credentialsValues = [
+			{
+				displayName: 'no credential found',
```

Comment:
please create resource strings for the new user strings by adding them to a constant in the 'strings.ts' file

Reply:
pushed a new commit to fix